de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/types/category/ABeanCategoryAssignment.java

```diff
@@ -53,6 +55,11 @@ public void setATypeInstance(ATypeInstance ti) {
 		helper.setCategoryAssignment(this.ti);
 	}
 	
+	@Override

```

Comment:
Do we still need this? I think this is just a fragment that stuck around from when phillip and me were playing around tih the annotations on category assignemnt level.

Reply:
You are right, removed